Enrich_methods/example.py

```diff
@@ -0,0 +1,210 @@
+import material_db_tools as mdbt
+from material_library import (
+    mat_data as pure,
+)  # this is importing the material dictionaries
+
+mat_lib_test = MaterialLibrary()
+# you may need to adjust this path for your setup
+mat_lib_test.from_json("material_library.json")
+
+
+simulation_lib = MaterialLibrary()
+
+
+################################### Method 3
+
+
+# Currently how Edgar and I are enriching out materials
+def enriched_lithium(enrichment):
+    enriched_li = Material({"Li6": enrichment, "Li7": 1 - enrichment})
+    return enriched_li
+
+
+def Li2TiO3Li_mat(enrichment=0.60):
+    Li2TiO3Li60 = Material()
+    Li2TiO3Li60.from_atom_frac(
+        {enriched_lithium(enrichment): 2, 80000000: 3, 220000000: 1}
+    )
+    Li2TiO3Li60.density = 3.43
+    Li2TiO3Li60 = Li2TiO3Li60.expand_elements()
+    Li2TiO3Li60.metadata["citation"] = "HernandezFusEngDes_2018"
+    return Li2TiO3Li60
+
+
+def Li4SiO4Li_mat(enrichment=0.60):
+    Li2TiO3Li60 = Material()
+    Li2TiO3Li60.from_atom_frac(
+        {enriched_lithium(enrichment): 4, 80000000: 4, 140000000: 1}
+    )
+    Li2TiO3Li60.density = 2.40
+    Li2TiO3Li60 = Li2TiO3Li60.expand_elements()
+    Li2TiO3Li60.metadata["citation"] = "HernandezFusEngDes_2018"
+    return Li2TiO3Li60
+
+
+# I also need to have another version of this function for the pin breeder zone.
+# I will if need to either make a version of this for each material I'm testing
+# or I will need a universial function that will require me to pass all the arugments in hcpb_pin_bw()
+# plus whatever material I'm testing for the case. Currently I'm testing around 8 different materials
+# How I make the universal function will depend on how I can import the library or if I can use enrich
+def hcpb_pin_bw(
+    enrichment=0.6,
+    Li4SiO4_frac=0.065,
+    Li2TiO3_frac=0.035,
+    he_frac=0.482,
+    EUROFER97_frac=0.418,
+):
+
+    Li4SiO4Li = Li4SiO4Li_mat(enrichment)
+    Li2TiO3Li = Li2TiO3Li_mat(enrichment)
+
+    mix = MultiMaterial(
+        {
+            mat_lib_test["EUROFER97"]: EUROFER97_frac,
+            Li4SiO4Li: Li4SiO4_frac,
+            mat_lib_test["HeT410P80"]: he_frac,
+            Li2TiO3Li: Li2TiO3_frac,
+        }
+    )
+    hcpb_pin_bw = mix.mix_by_volume()
+    hcpb_pin_bw.metadata["constituentcitation"] = "HernandezFusEngDes_2018"
+    return hcpb_pin_bw
+
+
+################################### Method 2
+
+# Method if you want to use enrich function without it being called in mix_by_volume or in make_mat_from_atom
+# But this version uses the output json file which will give you the same answer as rebuiling the material to the second last decimal
+# Earlier you said that you wanted to avoid that method since you don't trust it
+enriched_mat = mdbt.Material({30060000: 0.6, 30070000: 0.4})
+collapsed = mat_lib_test["Li2TiO3nat"].collapse_elements({3})
+atom_frac_enriched = mdbt.enrich(
+    collapsed.to_atom_frac(), 30000000, enriched_mat.to_atom_frac()
+)
```

Comment:
I see how the need to collapse from the fully processed library is cumbersome - I agree that this not ideal.

Reply:
I also see that converting between mass enrichment and atom fractions is cumbersome, but could be done in a single line:
```
enriched_atom_fracs = mbdt.Material({'Li6' : 0.6, 'Li7' : 0.4}).to_atom_frac()
```

But there is the danger that the user doesn't recognize the risk of missing atom and mass fractions